frontend/src/components/Cards/Card/Index.jsx

```diff
@@ -1,9 +1,11 @@
+import "../../../services/reset.css";
+import PropTypes from "prop-types";
 import "./style.css";
 
-export default function Card() {
+export default function Card({ hoverable }) {
   return (
     <section className="frontImage">
-      <div className="card">
+      <div className={`card ${hoverable ? "hoverable" : ""}`}>
```

Comment:
You told in propTypes that `hoverable` is required -> no need to test it's existence